reference/conanfile/attributes/binary_model.inc

```diff
@@ -74,9 +74,31 @@ If you want to do a safe check of settings values, you could use the ``get_safe(
         # Will be the default version if the return is None
         build_type = self.settings.get_safe("build_type", default="Release")
 
-The ``get_safe()`` method will return ``None`` if that setting or sub-setting doesn't
+The ``get_safe()`` method returns ``None`` if that setting or sub-setting doesn't
```

Comment:
What about adding some link here in the introduction to the settings.yml and mentioning that these values come from that file?